recipe/meta.yaml

```diff
@@ -39,23 +36,350 @@ requirements:
+    requirements:
+      build:
+        - posix                              # [win]
+        - {{ posix }}autoconf                  # [win]
+        - {{ posix }}automake{{ am_version }}  # [win]
+        - {{ posix }}libtool
+        - {{ posix }}patch                     # [win]
+        - {{ posix }}sed                       # [win]
+        - {{ native }}binutils                 # [win]
+        - {{ native }}crt-git                  # [win]
+        - {{ compiler('c') }}                # [not win]
+        - {{ compiler('cxx') }}              # [not win]
+        - make                               # [unix]
+      host:
+        - libiconv  # [not linux]
+        - {{ pin_subpackage('libintl', exact=true) }}
+      run:
+        - {{ pin_subpackage('libintl', exact=true) }}
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libintl${SHLIB_EXT}                   # [osx]
+        - if not exist %PREFIX%\Library\lib\intl.dll.lib exit /b 1    # [win]
+    about:
+      home: https://www.gnu.org/software/gettext/
+      license: LGPL-2.1-or-later
+      license_file: COPYING
+      summary: library that provides native language support to programs
+  - name: libasprintf
+    build:
+      run_exports:
+        # Seems pretty good: https://abi-laboratory.pro/tracker/timeline/gettext/
+        - {{ pin_subpackage('libasprintf') }}
+      ignore_run_exports:
+        - libcxx
+        - libiconv
+    script: install-asprintf.sh  # [unix]
+    script: install-asprintf.bat  # [not unix]
+    requirements:
+      build:
+        - posix                              # [win]
+        - {{ posix }}autoconf                  # [win]
+        - {{ posix }}automake{{ am_version }}  # [win]
+        - {{ posix }}libtool
+        - {{ posix }}patch                     # [win]
+        - {{ posix }}sed                       # [win]
+        - {{ native }}binutils                 # [win]
+        - {{ native }}crt-git                  # [win]
+        - {{ compiler('c') }}                # [not win]
+        - {{ compiler('cxx') }}              # [not win]
+        - make                               # [unix]
+      host:
+        - libiconv  # [not linux]
+    test:
+      commands:
+        # Only the actual binary (versioned) version should be in this package
+        - test -f ${PREFIX}/lib/libasprintf.*.dylib                   # [osx]
+        - test -f ${PREFIX}/lib/libasprintf.so.0                      # [linux]
+        - if not exist %PREFIX%\Library\bin\asprintf-*.dll exit /b 1  # [win]
+        # The unversioned variants as well as the lib on Windows should be in the main gettext package.
+        - test ! -f ${PREFIX}/lib/libasprintf$SHLIB_EXT               # [not win]
+        - if exist %PREFIX%\Library\lib\asprintf.dll.lib exit /b 1    # [win]
+    about:
+      home: https://www.gnu.org/software/gettext/
+      license: LGPL-2.1-or-later
+      license_file: COPYING
+      summary: Decomposed printf argument list
```

Comment:
This summary is a bit odd; one alternative that I can find is: "C-style formatted output in C++"